Bottom: feea54e910b879f05530643831541b1f8e9e4093
Top:    d313f10735cb33598e1e326ef8dde77756e38665
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-13 08:21:43 +0800

Refresh of tuntap-tweak-on-the-path-of

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index d425830..2560378 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1621,7 +1621,7 @@ static u32 tun_do_xdp(struct tun_struct *tun,
 			break;
 		goto out;
 	case XDP_TX:
-		*err = tun_xdp_xmit(tun->dev, xdp);
+		*err = tun_xdp_tx(tun->dev, xdp);
 		if (*err)
 			break;
 		tun_xdp_flush(tun->dev);
